deviceplugin/path.go

```diff
@@ -0,0 +1,114 @@
+// Copyright YEAR the generic-device-plugin authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deviceplugin
+
+import (
+	"crypto/sha1"
+	"fmt"
+	"k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
```

Comment:
Let's keep the imports that aren't from the stdlib grouped separately

Reply:
Resolved 3927c05 (I thought I had my linter on to catch these, whoops)